### DISCUSSION TITLE
Latest version busted....

### DISCUSSION THREAD

**Discussion by scrubdev1421** [farag2/Sophia-Script-for-Windows#583] [General]
`./sophia.ps1: File E:\Windows_Stuff\Scripts\Sophia.Script.for.Windows.11.PowerShell.7.v6.6.8\Sophia_Script_for_Windows_11_PowerShell_7_v6.6.8\Sophia.ps1 cannot be loaded. The contents of file E:\Windows_Stuff\Scripts\Sophia.Script.for.Windows.11.PowerShell.7.v6.6.8\Sophia_Script_for_Windows_11_PowerShell_7_v6.6.8\Sophia.ps1 might have been changed by an unauthorized user or process, because the hash of the file does not match the hash stored in the digital signature. The script cannot run on the specified system. For more information, run Get-Help about_Signing..`

**Comment by farag2** [comment]
https://github.com/farag2/Sophia-Script-for-Windows?tab=readme-ov-file#how-to-use

**Comment by farag2** [comment]
![изображение](https://github.com/farag2/Sophia-Script-for-Windows/assets/10544660/408916d0-701b-40c4-a44b-5cc0ac02fc4e)

This cert should be added as a trusted one, so by default you won't be allowed to run Sophia Script with `AllSigned` policy enabled as Sophia Script cert isn't a trusted one, nor generated locally, but in cloud. Please use `Set-ExecutionPolicy -ExecutionPolicy Bypass -Force` (RemoteSigned)
